src/client/theme-default/styles/components/vp-doc.css

```diff
@@ -108,12 +109,16 @@
 
 .vp-doc a {
   font-weight: 500;
-  color: var(--vp-c-brand);
+  color: var(--vp-c-brand-1);
+  text-decoration: underline dotted;
+  text-underline-offset: 2px;
```

Comment:
```suggestion
  text-decoration: underline;
  text-underline-offset: calc(0.25em + 1px);
```

This looks cleaner IMO:

<img width="750" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/ab956013-c7c3-4968-8b3f-75b9a5013c0d">

Reply:
Looking good! I like the second one (`0.125em`) better 👀